OutOfSchool/OutOfSchool.WebApi.Tests/Services/ApplicationServiceTests.cs

```diff
@@ -699,7 +700,11 @@ public async Task UpdateApplication_WhenIdIsValid_ShouldReturnApplication()
         applicationRepositoryMock.Setup(a => a.Count(It.IsAny<Expression<Func<Application, bool>>>())).ReturnsAsync(int.MinValue);
 
         // Act
-        var result = await service.Update(update, Guid.NewGuid()).ConfigureAwait(false);
+        var response = await service.Update(update, Guid.NewGuid()).ConfigureAwait(false);
+        Result<ApplicationDto> result = new Result<ApplicationDto>();
+        response.Match<Result<ApplicationDto>>(
+            actionResult => result = new Result<ApplicationDto>(),
+            succeed => result = succeed);
```

Comment:
(Excessive code) Please use `.Match` return value to write shorter and cleaner code. I.e.
` 
        var result = response.Match(errResult => new Result<ApplicationDto>(), result => result);
`

Reply:
Done